.github/workflows/publish_to_pypi.yml

```diff
@@ -1,6 +1,8 @@
 name: Publish django-guid to PyPI and TestPyPI 📦
```

Comment:
Seems like the possibility to push `rc` is gone now? So there's no way of publishing to test PyPi specifically?

Reply:
Not that I really need it I guess

Reply:
I've been thinking about removing test-pypi as a workflow in other projects, and just pushing the test-version locally before publishing a release. I know we generally want to replicate the "prod" job and environment when running a test, but in this case, the test releases have only ever failed for me when the package contents contain something weird, like badly formatted rst. I don't have a strong opinion about this, but what do you think of cutting the test-job altogether?

Reply:
Yeah, that works for me too honestly.  
Need to update `docs/publish.rst` if it's removed though.

Reply:
Updated docs and removed test-section of workflow.